code/modules/mob/mob.dm

```diff
@@ -1455,6 +1455,8 @@
 
 /mob/proc/update_equipment_speed_mods()
 	var/speedies = equipped_speed_mods()
+	if(speedies > 0 && HAS_TRAIT(src, TRAIT_SETTLER)) //if our movespeed mod is in the negatives, we don't modify it since that's a benefit
+		speedies *= 0.2
```

Comment:
The rise of the Big Slappy + Settler meta...
But truthfully, that's way too much of a combat advantage. It should be a tad more modest if you ask me.

Reply:
You know i only discovered the Big Slappy like, four hours after I made this PR.

Either way, the settler will still be slow on-foot compared to others, just faster with heavier loads.

Reply:
You're also right.

Reply:
If this is mostly alright then, I'll resolve this but honestly if people use settler too much to get away with things like hauling around big slappy, I'll come back to it